Resources/StyleGuide.md

```diff
@@ -0,0 +1,60 @@
+   <p><span style="background-color: rgba(249, 168, 212, 0.5)">As of the Minecraft Beta 1.2 update on January 13, 2011, there are now four kinds of wood.</span><span style="background-color: rgba(252, 211, 77, 0.5)">One is the normal wood (oak), another resembles the wood of silver birch trees, yet another type resembles the normal wood, but it is darker and appears in pine/conifer trees that grow in colder biomes, the fourth type is similar to the oak wood, however there are some color differences, and it is tilted to one side.</span> Wood blocks produce 4 wooden planks when crafted. Wood from different types of trees do not stack in the inventory. <span style="background-color: rgba(249, 168, 212, 0.5)">Planks made from different kinds of trees used to be completely identical</span>. Birch trees have slightly duller colored leaves than regular trees, pine trees have pine needles, and jungle leaves are leafy with fruit looking shapes on them.</p>
+   The fourth type of wood <span style="background-color: rgba(249, 168, 212, 0.5)">was introduced in snapshot 12w03a</span>, solely occurring in jungle biomes, and comprising trees exclusive to them. The tallest trees have this type of wood in 2x2 dimensions instead of the normal 1x1.
+</div>
+<br>
+
+## Capitalization
+If there is only one of a thing in intended to be in a world then it is a proper noun and should be capitalized. For example: Underground, there are randomly generated _mineshafts_. A _desert pyramid_ contains some rare loot. _Blazes_ spawn in _nether fortresses_; a _Nether_ structure. In _deep ocean_ biomes, _monuments_ can generate. A _stronghold_ is home to an _end portal_ that takes you to the _End_.
+
+## Images
+- Screenshots that use custom texture packs, UI mods and other custom content should not be used.
+- Only one image should be used for each individual attribute of the article.
+- Images should be dedicated to one attribute; showing parts of bigger contraptions should be avoided.
+- Images should not have the sole purpose of showcasing a bug, instead link to the bug on the [official bug tracker](https://bugs.mojang.com/secure/Dashboard.jspa).
+
+## Coordinates
+- Single in-game coordinates should be capitalized and unspaced ("Y=1" instead of "y=1" or "y = 1").
+- Coordinates should be in the order X, Y, Z with commas delimiting the individual coordinates.
+- Volumes should be in the order X, Y, Z, with each item separated by a multiplication sign ("×"; &times;); "4×3×2" is an area that is 4 blocks wide along the X axis, 3 along the Y axis (vertical), and 2 along the Z axis.
+
+## Commands 
+- Commands should be in code blocks made with backticks \`.
+- Literal keywords that must be typed in chat do not have any brackets for formatting applied (e.g., `/data merge`).
+- A list of valid keywords should be placed in parentheses with each option separated by a pipe (e.g., (get|merge|set|remove).
+- Variables must be inside angle brackets (e.g., `\<target\>`).
```

Comment:
```suggestion
- Variables must be inside angle brackets (e.g., `headYaw`).
```

Reply:
I have it set up this way becuase this is how Minecraft suggests commands and how most Unix clis write their help pages

Reply:
two lines above that you didn't write `/<command> <argument>`

Reply:
That's becuase on line 52 it's a mock article is talking about the data merge command not the data one